README.rst

```diff
@@ -58,6 +57,7 @@ Add the migration linter your ``INSTALLED_APPS``:
 ``--database DATABASE``                            Specify the database for which to generate the SQL. Defaults to *default*.
 ``--cache-path PATH``                              specify a directory that should be used to store cache-files in.
 ``--no-cache``                                     Don't use a cache.
+``--project-root-path=DJANGO_PROJECT_FOLDER``      An absolute or relative path to the django project.
```

Comment:
``--project-root-path DJANGO_PROJECT_FOLDER``  to stay consistant ;) (with space, not =)